tasmota/tasmota_xdrv_driver/xdrv_65_tuyamcubr.ino

```diff
@@ -0,0 +1,916 @@
+/*
+ * xdrv_65_tuyamcubr.ino - TuyaMCU Bridge support for Tasmota
+ */
+
+/*
+ * Copyright (C) 2023 David Gwynne <david@gwynne.id.au>
+ *
+ * Permission to use, copy, modify, and distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#ifdef USE_TUYAMCUBR
+
+/*
+ * Tuya MCU Bridge
+ */
+
+#define XDRV_65			65
+
+#ifndef nitems
+#define nitems(_a)		(sizeof((_a)) / sizeof((_a)[0]))
+#endif
+
+#ifndef CTASSERT
+#define CTASSERT(x)		extern char  _ctassert[(x) ? 1 : -1 ]	\
+				    __attribute__((__unused__))
+#endif
+
+#define TUYAMCUBR_LOGNAME		"TuyaMCUBr"
```

Comment:
Pls shorten log name to three character in line with Tasmota logging. Suggesting "TYB"